presentation/slides.md

```diff
@@ -1,18 +1,99 @@
 ---
-theme: seriph
-background: https://source.unsplash.com/collection/94734566/1920x1080
-class: 'text-center'
-highlighter: shiki
-lineNumbers: false
-info: |
-  Presentation slides for developers.
-  Learn more at [Sli.dev](https://sli.dev)
+theme: apple-basic
+layout: intro
+colorSchema: 'dark'
 ---
 
-# Welcome to Slidev
+# Cast: Podcast Listening
 
-Hello world
+Innopolis University •
+Spring 2022 •
+Software Quality and Reliability
 
---- 
+<p class="absolute bottom-0 text-sm">
+  AbdelRahman Abounegm •
+  Khaled Ismaeel •
+  Lev Chelyadinov •
+  Magomed Magomedov •
+  Max Korsunov
+</p>
 
-# Page 2
+---
+layout: fact
+---
+
+# "Cast"
+
+Your go-to podcast listening app.  
+Simple yet powerful, works anywhere.
+
+---
+layout: image-right
+image: "img/leo-wieling-bG8U3kaZltE-unsplash.jpg"
+---
+
+# Key Features
+
+- 🔊 Listen to podcasts
+- 💾 Download for listening offline
+- 📝 Read transcriptions
+- 🚪 Keep data in personal profile (**Auth0**)
+- 👍 Write reviews & comments for podcasts
+- 📱 Use on any device with browser (**PWA**)
+
+<p class="absolute bottom-10 text-sm">Cast • 2</p>
+
+---
+layout: image-right
+---
+Sprint #0: Progress
+
+- 🫂 Team building & setting up collaboration
```

Comment:
The icon here appears as a box with a question mark for me. Could it be some emoji added only in Windows 11?

Reply:
Possibly.